src/powershell/Public/Remove-FinOpsHubScope.ps1

```diff
@@ -0,0 +1,82 @@
+
+    .PARAMETER HubName
+    Optional. Name of the hub instance to update.
+
+    .PARAMETER RemoveData
+    Optional. Indicates whether to remove data for this scope from storage. Default = false
+
+    .EXAMPLE
+    Remove-FinOpsHubScope -Id "/providers/Microsoft.Billing/billingAccounts/123" -HubName "FooHub" 
+    Deletes the exports configured to use the FooHub hub instance. Existing data is retained in the storage account.
+    
+    Remove-FinOpsHubScope -Id "/subscriptions/##-#-#-#-###" -HubName "FooHub" -RemoveData
+    Deletes the exports configured to use the FooHub hub instance and removes data for that scope.
+    
+    .LINK
+    https://aka.ms/ftk/Remove-FinOpsHubScope
+#>
+
+
+function Remove-FinOpsHubScope {
+    [CmdletBinding(SupportsShouldProcess)]
+    param (
+        [Parameter(Mandatory = $true)]
+        [string]
+        $Id,
+        [Parameter()]
+        [string]
+        $HubName,
+        [Parameter()]
+        [switch]
+        $RemoveData
+    )
+
+    try {
+        
+
+        $hub = Get-FinOpsHub -Name $HubName
+        $storageAccountId = $hub.StorageAccountId
+
+        #Get all exports for the scope that are pointed to the included storage account and the msexports container
+        $exports = Get-FinOpsCostExport -Scope $Id | Where-Object { $_.StorageAccountId -eq $storageAccountId } 
+
+        # Delete the exports
+        foreach ($export in $exports) 
+        {
+            if($PSCmdlet.ShouldProcess("Cost Management Export","Delete"))
+            {
+                Write-Verbose -Message "Deleting Cost Management export $($export.Name) from storage account $($export.StorageAccountId.Split("/")[-1])."
+                Remove-FinOpsCostExport -Scope $Id -Name $export.Name -RemoveData:$RemoveData
+                Write-Verbose -Message "Complete: Deleted Cost Management export $($export.Name) from storage account $($export.StorageAccountId.Split("/")[-1])."
+            }
+
+            # Delete the data if requested
+            if ($RemoveData) 
+            {
+                # This can use the standard storage Az commands
+                if($PSCmdlet.ShouldProcess("Cost Management Export Data","Delete"))
+                {
+                    
+                    $storageAccountName = $export.StorageAccountId.Split("/")[-1]
+                    $storageAccountKey = (Get-AzStorageAccountKey -ResourceGroupName (Get-AzResource -ResourceId $export.StorageAccountId).ResourceGroupName -Name $storageAccountName).Value[0]
+                    $context = New-AzStorageContext -StorageAccountName $storageAccountName -StorageAccountKey $storageAccountKey
+                    Write-Verbose -Message "Deleting data for Cost Management export $($export.Name) in storage account $($export.StorageAccountId.Split("/")[-1]) at path $($export.StoragePath)."
+                    Remove-AzDataLakeGen2Item -FileSystem "ingestion" -Path $export.StoragePath -Context $context -Force
```

Comment:
Use the scope instead of the export destination. They're supposed to be the same, but may not be since we move files around in ADF.
```suggestion
                    Remove-AzDataLakeGen2Item -FileSystem "ingestion" -Path $scope -Context $context -Force
```

Reply:
I think we need $export.StoragePath $export.Id i.e., "subscriptions/927d8034-e8db-4ddb-9e3f-4c085733b991/providers/Microsoft.CostManagement/exports/ftk-costexport" does not contain storage path.

Reply:
You need the part before `/providers/Microsoft.CostManagement/exports/`. In your case, it's ""subscriptions/927d8034-e8db-4ddb-9e3f-4c085733b991". That's the path we store files in. We don't use the export storage path.

Reply:
```suggestion
                    Remove-AzDataLakeGen2Item -FileSystem "ingestion" -Path $export.Id.ToLower()Split("/provider/microsoft.costmanagement/exports/")[0] -Context $context -Force
```